### ISSUE TITLE
about UNet on Houston2018 data set

### ISSUE THREAD

**Issue by kkkkkkb** [yyyyangyi/Multi-source-RS-DGConv#3]
Hi, @yyyyangyi
Thanks for your work and code!
I have tried to experiment with SepG-UNet on Houston2018 data set,but OA was only 59.67±1.46%.
Here are some codes within common.py.
`    use_gpu = True
    use_dgconv = True
    fix_groups = 1     
    
    num_replicates = 5
    seed = 42
    dataset = 'houston' `

I have also experimented with UNet on Houston2018 data set,but OA was only 59.28±0.92%.
Here are some codes within common.py.
`    use_gpu = True
    use_dgconv = False
    fix_groups = 1     
    
    num_replicates = 5
    seed = 42
    dataset = 'houston' `

And I have also experimented with SepG-ResNet18 on Berlin data set,but OA was only 64.06±4.17%.
Here are some codes within common.py.
`    use_gpu = True
    use_dgconv = True
    fix_groups = 1     
    
    num_replicates = 5
    seed = 42
    dataset = 'berlin' `

Could you please tell me anything wrong are there?

Thanks a lot!

**Comment by yyyyangyi** [comment]
Hi, 

Thank you for your interest in our work. We believe the most possible reason is that you are training your model in an environment that is different from ours (we used NVIDIA GTX 1080 Ti GPUs and PyTorch 1.9.0, please see our paper Section IV. B on page 8 for more details of our experimental setup). 

Completely reproducible results are NOT guaranteed across different software/hardware, which is a known issue with PyTorch (please see [this page](https://pytorch.org/docs/stable/notes/randomness.html) for more details). In our experiments we actually reproduced baseline models Fusion-FCN and TB-CNN, and that's why we put our implementation of those in this repository. 

You probably need to tune the hyperparameters on your device to obtain optimal performance. However, if you stick to our configurations and try to reproduce our ablation analysis as discussed in our paper Section IV. C, you should get a performance curve showing a very similar trend as our Fig. 9.

Hope this helps!

**Comment by kkkkkkb** [comment]
Thank you,it's very kind of you!

**Comment by kkkkkkb** [comment]
> Hi,
> 
> Thank you for your interest in our work. We believe the most possible reason is that you are training your model in an environment that is different from ours (we used NVIDIA GTX 1080 Ti GPUs and PyTorch 1.9.0, please see our paper Section IV. B on page 8 for more details of our experimental setup).
> 
> Completely reproducible results are NOT guaranteed across different software/hardware, which is a known issue with PyTorch (please see [this page](https://pytorch.org/docs/stable/notes/randomness.html) for more details). In our experiments we actually reproduced baseline models Fusion-FCN and TB-CNN, and that's why we put our implementation of those in this repository.
> 
> You probably need to tune the hyperparameters on your device to obtain optimal performance. However, if you stick to our configurations and try to reproduce our ablation analysis as discussed in our paper Section IV. C, you should get a performance curve showing a very similar trend as our Fig. 9.
> 
> Hope this helps!

Thank you,it's very kind of you!
